indexer/CMakeLists.txt

```diff
@@ -30,6 +30,8 @@ set(
   complex/serdes.hpp
   complex/serdes_utils.hpp
   complex/tree_node.hpp
+  contacts_processor.cpp
+  contacts_processor.hpp
```

Comment:
validate_and_format_contacts would be a better name.

Reply:
renamed to `validate_and_format_contacts`